gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -62,10 +62,42 @@ <h1 class="title">{{ vulnerability.id }}</h1>
           <dd>{{ vulnerability.published }}</dd>
           <dt>Modified</dt>
           <dd>{{ vulnerability.modified }}</dd>
+          {%- if vulnerability.severity -%}
+          <dt>Severity</dt>
+          <dd>
+            <ul class="aliases">
+              {% for item in vulnerability.severity -%}
+              <li>{{ item.type }} - {{ item.score }}</li>
+              {% endfor -%}
+            </ul>
+          </dd>
+          {%- endif -%}
           <dt>Details</dt>
           <dd class="details">
             {{ vulnerability.details|markdown|safe -}}
           </dd>
+          {% if vulnerability.credits -%}
+          <dt class="credits">Credits</dt>
+          <dd class="credits">
+            <ul>
+              {% for credit in vulnerability.credits -%}
+              <li class="credit">
+                <ul>
+                  <li>{{ credit.name }}{%- if 'type' in credit -%} - {{ credit.type }}{{%- endif -%}}</li>
+                  {%- if 'contact' in credit -%}
+                  <li>
+                    <ul class="contact">
+                      {%- for item in credits.contact -%}
+                      <li><a href="{{ item }}" target="_blank" rel="noopener noreferrer">{{ item }}</a></li>
```

Comment:
I'm seeing this:
```
  File "/usr/local/google/home/ochang/.local/share/virtualenvs/appengine-cJoj1_Ix/lib/python3.11/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/usr/local/google/home/ochang/.local/share/virtualenvs/appengine-cJoj1_Ix/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/usr/local/google/home/ochang/osv/gcp/appengine/dist/vulnerability.html", line 86, in template
    <li>{{ credit.name }}{%- if 'type' in credit -%} - {{ credit.type }}{{%- endif -%}}</li>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.TemplateSyntaxError: unexpected '%'
```

With trying this on `/vulnerability/GO-2024-2824`.

Could you please fix?

Reply:
Oops! Yes, this was due to an oversight on the template tag setup. It should have been resolved now:

<img width="1163" alt="Screenshot 2024-05-09 at 7 39 52 PM" src="https://github.com/google/osv.dev/assets/73332835/934727b1-c759-44dd-87bc-42e58041ca52">